odk1-src/form-update.rst

```diff
@@ -28,8 +28,12 @@
   las
   th
 
-Updating forms
-=================
+Updating forms in Aggregate
+============================
+
+.. note::
+
+  This page is only about ODK Aggregate. ODK Central allows any kind of form update that doesn't change the type of a field. See :ref:`central-forms-updates`.
```

Comment:
I don't think it's throwing shade. It's an important restriction and easy to avoid if known ahead of time. It's irritating to learn about when attempting it.